man/bundle-init.ronn

```diff
@@ -12,7 +12,17 @@ adding a [`Gemfile(5)`][Gemfile(5)] to a gem with a gemspec, the `--gemspec` opt
 automatically add each dependency listed in the gemspec file to the newly
 created [`Gemfile(5)`][Gemfile(5)].
 
+One of the lines included in the default [`Gemfile(5)`][Gemfile(5)]
```

Comment:
This should be in it's own section as we're no talking longer about the command itself.